### PR TITLE
[cluster-api Azure] Add pipeline and script to create new marketplace sku

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
Further automate publishing new images by adding a release pipeline to create new disk versions and a new pipeline for creating SKUs. Using https://github.com/devigned/pub to script Azure Cloud Partner Portal operations.

### CODE REVIEW

File: images/capi/packer/azure/scripts/new-sku.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+patch=${ADDR[2]}
+
+sku_id="k8s-${major}dot${minor}dot${patch}-ubuntu-1804"
+
+< $SKU_TEMPLATE_FILE sed s/{{ID}}/"$sku_id"/ | sed s/{{K8S_VERSION}}/"$K8S_VERSION/" > sku.json
+cat sku.json
+
+echo
+echo "Getting pub..."
+(set -x ; curl -fsSL https://github.com/devigned/pub/releases/download/v0.2.0/pub_v0.2.0_linux_amd64.tar.gz -o pub; tar -xzf pub)
```
Comment (marosset) [inline images/capi/packer/azure/scripts/new-sku.sh]: If we have to update the version of pub again we should consider moving the version to a pipeline variable.

Comment (CecileRobertMichon) [reply images/capi/packer/azure/scripts/new-sku.sh]: I think it makes sense to keep it in source control and update the version with a PR

Comment (marosset) [reply images/capi/packer/azure/scripts/new-sku.sh]: Yea, i guess it won't change that often.

**Comment by devigned** [comment]
Yay, no more UI! lgtm

**Pull request merged**